### ISSUE TITLE
Macforge plugins don't take effect until opening the Macforge GUI (tested with CleanHUD only)

### ISSUE THREAD

**Issue by samaero** [w0lfschild/MacForge#33]
After the GUI has been opened once, the app can be quit and CleanHUD persists, which makes me think the helper may not be being loaded correctly in the background.

**Comment by w0lfschild** [comment]
If you add the helper manually to your login items does it work?

**Comment by samaero** [comment]
Yep, that seems to have done it - thanks! The helper was already added, so I had to remove it from the login items, hit Show Package Contents on the app, create a shortcut to MacForgeHelper on the desktop, and add it from there.
